src/seedu/addressbook/data/AddressBook.java

```diff
@@ -121,6 +121,13 @@ public UniqueTagList getAllTags() {
         return new UniqueTagList(allTags);
     }
 
+    /**
+     * Sort the address book in an alphabetical order
+     */
+    public void sortAllPersons(){
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout) violation! One more more of these problems detected:
1. extra/missing spaces.
2. brace style problem.
3. missing braces.
4. name doesn't follow naming convention